client/my-sites/domains/domain-management/edit/card/notices/expiring-soon.jsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { useLocalizedMoment } from 'components/localized-moment';
+
+/**
+ * Internal dependencies
+ */
+import { isExpiringSoon } from 'lib/domains/utils';
+import RenewButton from 'my-sites/domains/domain-management/edit/card/renew-button';
+import { type as domainTypes } from 'lib/domains/constants';
+
+function ExpiringSoon( props ) {
+	const translate = useTranslate();
+	const moment = useLocalizedMoment();
+
+	const { domain, purchase, selectedSite } = props;
+	const { expiry } = domain;
+
+	if ( ! isExpiringSoon( domain, 30 ) ) {
+		return null;
+	}
+
+	let noticeText;
+	let subscriptionId;
+	let customLabel;
+	let tracksProps;
+
+	if ( domain.type === domainTypes.MAPPED ) {
+		if ( ! domain.currentUserCanManage ) {
+			noticeText = translate(
+				'{{strong}}The domain mapping will expire{{/strong}} in {{strong}}%(days)s{{/strong}}. Please contact the domain mapping owner %(owner)s to renew it.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( '{{strong}}The domain will expire{{/strong}} in {{strong}}%(days)s{{/strong}}. Please contact the domain owner %(owner)s to renew it.' )` *ES Score: 8*